gremlin-javascript/glv/PackageJson.template

```diff
@@ -33,7 +33,10 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "ws": "^3.0.0"
+    "ws": "^3.0.0",
+    "grunt": "~1.0.2",
```

Comment:
These 3 dependencies should be added under `devDependencies` instead

Reply:
ok - fixed